lib/components/molecules/AtomicCarousel/types/index.js

```diff
@@ -0,0 +1,10 @@
+// @flow
+
+export type Props = {
+  dots: Boolean,
```

Comment:
Why is everything a mandatory prop

Reply:
Didn't get this. Doesn't it need to be isRequired to make it mandatory?